connectors/src/connectors/index.ts

```diff
@@ -129,6 +131,51 @@ export const RESUME_CONNECTOR_BY_TYPE: Record<
   },
 };
 
+type BotToggler = (
+  connectorId: ModelId,
+  botEnabled: boolean
+) => Promise<Result<void, Error>>;
+
+const toggleBotNotImplemented = async (
+  connectorId: ModelId,
+  botEnabled: boolean
```

Comment:
you can remove that line

Reply:
Ok ! A bit surprised since it doesn't match the signature of BotToggler, but TSC does indeed not seem to complain (despite the boolean arg being required)